www/source/blog/2017-05-19-Builder.html.md

```diff
@@ -0,0 +1,101 @@
+
+# What Is Builder?
+
+If you've visited this site before today and clicked on [sign-in](https://app.habitat.sh/#/sign-in) or [search packages](https://app.habitat.sh/#/pkgs/core) you've actually already seen and used Builder, albeit a much earlier iteration, but the service has been live since last June. Builder performs three jobs for the Habitat community - it builds packages, hosts them, and helps you discover them.
+
+## Market Place & Artifact Repository (The Depot)
+
+The goal of our initial offering of Builder was to provide just enough functionality to fulfill discovery and an API for the Supervisor and Habitat CLI clients for storing and retrieving artifacts. Since these were the only two features available at launch, Builder colloquially became known as the "The Depot" after its artifact repository component.
+
+## Hosted Build Service
+
+In today's milestone release we finally enable the namesake feature of Builder: the ability to build packages from plan definitions from a GitHub repository and publish them back to the public Depot. This feature is now available for all packages in white-listed origins starting with the [core origin](https://app.habitat.sh/#/pkgs/core). If you're working on an open source project that you'd like to package for Habitat we'd love to add your origin to the white-list to enable build functionality for your packages too! Send an email to <humans@habitat.sh> to apply for access.
+
+# Why Build Builder?
+
+There are plenty of great build servers to choose from which cover every combination of hosting option, monetization model, and license agreement, so why build another build service? The answer lies within a principle of our packaging system itself.
+
+Habitat packages are **immutable**, **atomic**, and **isolated** - you cannot change their contents - the services they may contain execute in their own process environment and don't share filesystem resources; and they are either fully installed and available on their host machine or they are not. It is not possible through regular use of Habitat to have a partially installed or modified Habitat package installation. The immutability of these packages also extends to what they are linked to.
+
+## Dependency Linking
+
+Habitat packages dynamically link to explicitly defined dependencies and they will only ever link to the exact version of their dependency they were built against when the package was generated. This gives Habitat packages the feeling of static linking but significantly reduces compilation times by allowing you to download your dependencies as pre-built, GPG signed, checksummed packages and dynamically link to them.
```

Comment:
I'd replace "GPG-signed" with "cryptographically signed" (as we don't use GPG anymore for signing package artifacts)